htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -952,13 +952,15 @@ function(data) {
 			var discount = parseFloat($('option:selected', this).attr('data-discount'));
 			if (isNaN(discount)) { discount = parseFloat(jQuery('#idprodfournprice').attr('data-discount'));}
 
-			/* var tva_tx = $('option:selected', this).data('tvatx'); */
+			var tva_tx = $('option:selected', this).data('tvatx'); /* FIX21543 */
```

Comment:
Do not include the comment of the fix into code. This information is already tracked by git.

Reply:
thank you for the correction. i will be careful next time.